### ISSUE TITLE
The MetaMask Firefox plugin pollutes the console for all pages

### ISSUE THREAD

**Issue by Gutza** [MetaMask/metamask-docs#159]
Most times I open Firefox console I find two messages from the MetaMask plugin – one of them is a warning I do not care about (_MetaMask will soon stop reloading pages on network change_), and the other one is an actual error (_Source map error: Error: NetworkError when attempting to fetch resource_), which is also related to the MetaMask plugin.

**Comment by BboyAkers** [comment]
Thanks for the submission I'll look into it!

**Comment by BboyAkers** [comment]
@whymarrh I think these are just general warnings about the `updates` for the extension correct?

**Comment by whymarrh** [comment]
Yup these are extension related. I'll transfer this issue to the extension repo.
